Mathlib/Tactic/ToAdditive.lean

```diff
@@ -90,7 +90,7 @@ that have previously been labeled with `to_additive`.
 
 In the `mul_comm'` example above, `to_additive` maps:
 * `mul_comm'` to `add_comm'`,
-* `comm_semigroup` to `add_comm_semigroup`,
+* `comm_semigroup` to `AddCommSemigroup`,
```

Comment:
I guess all the docs in this file ought to be updated; happy to split that into a separate PR.

Reply:
@fpvandoorn